### PR TITLE
Use Checks From `diagram-js`

### PR DESCRIPTION
**Pull request by philippfromme**
* use `isConnection`, `isLabel`, `isRoot` and `isShape` checks wherever possible

---

Requires https://github.com/bpmn-io/diagram-js/pull/783
Closes https://github.com/bpmn-io/bpmn-js/issues/1901

### CODE REVIEW

File: lib/features/copy-paste/BpmnCopyPaste.js [hunk 1]
```diff
@@ -191,3 +191,9 @@ BpmnCopyPaste.$inject = [
   'eventBus',
   'moddleCopy'
 ];
+
+// helpers //////////
+
+function isLabelLike(descriptor) {
+  return descriptor.labelTarget;
```
Comment (nikku) [inline lib/features/copy-paste/BpmnCopyPaste.js:198]: Why not be explicit about the shape of the object ("must have a `labelTarget` property"):

```suggestion
  return 'labelTarget' in descriptor;
```

File: lib/util/LabelUtil.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ import {
 
 import { is } from './ModelUtil';
 
+import { isLabel } from 'diagram-js/lib/model';
```
Comment (nikku) [inline lib/util/LabelUtil.js:7]: Double import.

File: test/spec/features/copy-paste/BpmnCopyPasteSpec.js [hunk 3]
```diff
@@ -1015,7 +1017,7 @@ function integrationTest(elementIds) {
       });
 
       // (4) move all elements except root
-      modeling.moveElements(elementRegistry.filter(isRoot), { x: 50, y: -50 });
+      modeling.moveElements(elementRegistry.filter(element => !isRoot(element)), { x: 50, y: -50 });
```
Comment (nikku) [inline test/spec/features/copy-paste/BpmnCopyPasteSpec.js:1020]: :see_no_evil:

Comment (philippfromme) [reply test/spec/features/copy-paste/BpmnCopyPasteSpec.js:1020]: 🤡

File: lib/util/LabelUtil.js (hunk 2 above)
Comment (philippfromme) [reply lib/util/LabelUtil.js]: Not sure how to avoid this.

**Review by nikku — APPROVED** [review]

**Pull request merged**